app/models/setting/provisioning.rb

```diff
@@ -72,7 +72,7 @@ def self.default_settings
       ),
       self.set(
         'excluded_facts',
-        N_("Exclude pattern for all types of imported facts (rhsm, puppet e.t.c.). Those facts won't be stored in foreman's database. You can use * wildcard to match names with indexes e.g. macvtap*"),
+        N_("Exclude pattern for all types of imported facts (rhsm, puppet e.t.c.). Those facts won't be stored in foreman's database. You can use * wildcard to match names with indexes e.g. ignore* will filter out ignore, ignore123 as well as a::ignore or even a::ignore123::b"),
```

Comment:
Slightly unrelated, but `e.t.c.` is a weird abbreviation for `et cetera`.

Reply:
Since I haven't touched this string and tests are passing and this is ready to go, let me fix this either in rebase or in the upcoming: https://projects.theforeman.org/issues/26984

Reply:
I am going to rebase this actually, since I am touching the string and CI is not overloaded... it's Friday! :-)